include/bios_disk.h

```diff
@@ -84,8 +84,10 @@ void incrementFDD(void);
 
 #define MAX_DISK_IMAGES (2 + MAX_HDD_IMAGES)
 
-extern std::array<std::shared_ptr<imageDisk>, MAX_DISK_IMAGES> imageDiskList;
-extern std::array<std::shared_ptr<imageDisk>, MAX_SWAPPABLE_DISKS> diskSwap;
+// Merely pointers. The actual raw image objects backing these
+// arrays are owned are managed by the drive manager class.
```

Comment:
Remove "are owned", or change "are managed" to "and managed".

Reply:
fixed.